packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -327,13 +329,18 @@ export default class NewVm extends BaseComponent {
   // Actions ---------------------------------------------------------------------
 
   _reset = () => {
+    const { template } = this.state.state
     this._replaceState({
       bootAfterCreate: true,
       CPUs: '',
       cpuCap: '',
       cpuWeight: '',
       existingDisks: {},
       fastClone: true,
+      hvmBootFirmware:
+        get(() => template.boot.firmware) !== undefined
+          ? get(() => template.boot.firmware)
+          : '',
```

Comment:
I'm not sure this is the right place for this, it seems that similar work is done in `_initTemplate`.